osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/pom.xml

```diff
@@ -46,6 +79,78 @@
           </nonFilteredFileExtensions>
         </configuration>
       </plugin>
+        <!-- build helper maven plugin used for adding integration-test directories -->
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>build-helper-maven-plugin</artifactId>
+          <version>1.9.1</version>
+          <executions>
+            <!-- add integration-test source directory -->
+            <execution>
+              <id>add-integration-test-sources</id>
+              <phase>generate-test-sources</phase>
+              <goals>
+                <goal>add-test-source</goal>
+              </goals>
+              <configuration>
+                <sources>
+                  <source>src/integration-test/java</source>
+                </sources>
+              </configuration>
+            </execution>
+            <!-- add integration-test resource directory -->
+            <execution>
+              <id>add-integration-test-resources</id>
+              <phase>generate-test-resources</phase>
+              <goals>
+                <goal>add-test-resource</goal>
+              </goals>
+              <configuration>
+                <resources>
+                  <resource>
+                    <filtering>true</filtering>
+                    <directory>src/integration-test/resources</directory>
+                  </resource>
+                </resources>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+        <!-- maven surefire plugin for running unit tests -->
+        <plugin>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 7. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=xml%3AIndentCheck)